seed/static/seed/partials/analyses_table.html

```diff
@@ -12,6 +12,7 @@ <h2><i class="fa fa-bar-chart"></i> {$:: 'Analyses' | translate $}</h2>
                 <thead>
                     <tr>
                         <th translate>Analysis Name (User Defined)</th>
+                        <th translate></th>
```

Comment:
This code change and the others on this page were part of the merge conflict resolution.